TTS/tts/utils/text/cleaners.py

```diff
@@ -1,9 +1,8 @@
 """Set of default text cleaners"""
 
-# TODO: pick the cleaner for languages dynamically
```

Comment:
The same comment https://github.com/idiap/coqui-ai-TTS/blob/dev/TTS/tts/utils/text/tokenizer.py#L109